src/themes/teams/components/Icon/svg/icons/add.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default (
+  <svg viewBox="0 0 32 32" role="presentation" className="app-svg icons-add" focusable="false">
```

Comment:
These need to be processed to fit with all other icons.  @codepretty can you take a look at these changes and ensure it is going to align with #441?